sormas-backend/src/main/java/de/symeda/sormas/backend/region/GeoShapeProviderEjb.java

```diff
@@ -167,18 +166,19 @@ public GeoLatLon getCenterOfDistrict(DistrictReferenceDto district) {
 		return new GeoLatLon(polygonCenter.getX(), polygonCenter.getY());
 	}
 
-	@PostConstruct
-	private void loadData() {
-
-		String countryName = configFacade.getCountryName();
-		if (countryName.isEmpty()) {
-			logger.warn("Shape files couldn't be loaded, because no country name is defined in sormas.properties.");
-		} else {
-			loadRegionData(countryName);
-			loadDistrictData(countryName);
-		}
-		buildCountryShape();
-	}
+	/*
+	 * @PostConstruct
+	 * private void loadData() {
```

Comment:
@MateStrysewske I do not think that this code is used anymore, I commented because it breaks if we configure a country name. Do you remember its original intent or can I delete it?